### PR TITLE
Updated code review process documentation

### PR DESCRIPTION
**Pull request by Onager**
Updated code review process documentation.

### CODE REVIEW

File: process/Code review process.asciidoc [hunk 1]
```diff
@@ -105,180 +75,40 @@ git push
...
-* determines the codereview.appspot.com issue number from the corresponding
-review file in `./reviews/`
-* runs the linter
-* runs the unit tests
-* checks if your local clone is up to date with the upstream/master and updates
-if necessary by running: `git pull --rebase upstream master`
-* uploads the diff with upstream/master to codereview.appspot.com
-* mails the maintainers
+* https://github.com/log2timeline/l2tdocs/blob/master/process/Github%20code%20review.md[Github]
+* https://github.com/log2timeline/l2tdocs/blob/master/process/Rietveld%20code%20review.asciidoc[Rietveld (codereview.appspot.com)]
```
Comment (joachimmetz) [inline process/Code review process.asciidoc]: remove ?

File: process/Code review process.asciidoc [hunk 2]
```diff
@@ -105,180 +75,40 @@ git push
...
-[NOTE]
-The format of the review file is subject to change. There are ideas to add
-the diffbase and other code review relevant information.
-
+[preface]
+== Revision history
+[cols="1,1,1,5",options="header"]
+|===
+| Version | Author | Date | Comments
+| 0.0.1 | Joachim Metz | January 2016 | Initial version based on template.
```
Comment (joachimmetz) [inline process/Code review process.asciidoc:144]: add an entry

File: process/Code review process.asciidoc [hunk 3]
```diff
@@ -105,180 +75,40 @@ git push
...
+| Keywords: | Design proposal
+| Status: | Draft
+|===
 
-It currently consists of a single number e.g.
+[preface]
+== License
 ....
-285170043
+Copyright (C) 2016, log2timeline projects maintainers <log2timeline-maintainers@googlegroups.com>
```
Comment (joachimmetz) [inline process/Code review process.asciidoc]: 2016-2018

File: process/Github code review.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# GitHub command line code review
+
+There are two options to start a code review on Github:
+
+## Option 1: Use the review.py script from l2tdevtools:
```
Comment (joachimmetz) [inline process/Github code review.md]: I would change the order of the options starting with the Github native one first

File: process/Github code review.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Github [here](https://help.github.com/articles/requesting-a-pull-request-review/).
+
+If you don't know who to request, just use the reviewer suggested by Github.
+
+# Status checks
+Once your pull request has been created, automated checkers will run to on your changes to check for mistakes, or code
+that doesn't match the style guide. Review the [output from the tools](https://help.github.com/articles/about-status-checks/), 
+and make sure your pull request passes.
+
+Your pull request [can't be merged](https://help.github.com/articles/about-required-status-checks/) until the checkers
```
Comment (joachimmetz) [inline process/Github code review.md]: What about CodeFactor and duplicate code alerts? I opt we are a bit more verbose here about where we will be lenient.

File: process/Github code review.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+report everything is OK. 
+
+## Receiving Github reviews
+Your pull request will be [assigned](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/)
```
Comment (joachimmetz) [inline process/Github code review.md]: How does work when using the github UI?

File: process/Github code review.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+```
+PYTHONPATH=./l2tdevtools python ./l2tdevtools/tools/review.py --project_path=/path/to/the/project create-pr
+```
+
+## Option 2: Create a pull request using the Github web UI
+This process is documented by Github [here](https://help.github.com/articles/creating-a-pull-request/). 
+Once this is done, request a review from one of the project maintainers, as documented by 
+Github [here](https://help.github.com/articles/requesting-a-pull-request-review/).
+
+If you don't know who to request, just use the reviewer suggested by Github.
```
Comment (joachimmetz) [inline process/Github code review.md]: Add something about assigning the PR

File: process/Github code review.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+## Receiving Github reviews
+Your pull request will be [assigned](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) 
+to a project maintainer either by the script or the reviewer. The maintainer will review your code, and may push some changes to your branch and/or request that you make changes.
+
+After they've reviewed your code, make any necessary changes, and push them to your feature branch. After that, reply to the comments made by the
+reviewer, then [request a new review](https://help.github.com/articles/requesting-a-pull-request-review/) from the same reviewer.
+
+## Merging pull requests
+Once the pull request assignee is happy with the code, and all the status checks have passed, they'll merge your code into
```
Comment (joachimmetz) [inline process/Github code review.md]: code => proposed changes

**Review by joachimmetz — CHANGES_REQUESTED** [review]

File: process/Code review process.asciidoc (hunk 1 above)
Comment (Onager) [reply process/Code review process.asciidoc]: Done

File: process/Code review process.asciidoc (hunk 3 above)
Comment (Onager) [reply process/Code review process.asciidoc]: Done

File: process/Code review process.asciidoc (hunk 2 above)
Comment (Onager) [reply process/Code review process.asciidoc:144]: Done.

File: process/Github code review.md (hunk 4 above)
Comment (Onager) [reply process/Github code review.md]: Done

File: process/Github code review.md (hunk 7 above)
Comment (Onager) [reply process/Github code review.md]: Users who aren't part of the project can't assign issues or PRs. That's why this just says to request a review, and it's then the reviewer or maintainer's responsibility to assign the issue.

File: process/Github code review.md (hunk 5 above)
Comment (Onager) [reply process/Github code review.md]: This is fixed - we need to fix up the checkers, or turn them off if they aren't useful.

File: process/Github code review.md (hunk 6 above)
Comment (Onager) [reply process/Github code review.md]: The reviewer or a maintainer needs to assign the pull request.

File: process/Github code review.md (hunk 8 above)
Comment (Onager) [reply process/Github code review.md]: Done

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**